lang_ml/parsing/ast_ml.ml

```diff
@@ -59,6 +59,7 @@ type todo_category = string wrap
 type type_ =
   | TyName of name (* include builtins *)
   | TyVar of ident (* 'a *)
+  | TyAny of ident (* _ *)
```

Comment:
same here, TyAny of tok

Reply:
This was a bit lazy on my part. It would be clearer to only have a tok as argument.